packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -62,6 +62,15 @@ export interface EcsRunTaskProps extends sfn.TaskStateBaseProps {
    * @see https://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_types.html
    */
   readonly launchTarget: IEcsLaunchTarget;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service
+   *
+   * @see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RunTask.html#ECS-RunTask-request-propagateTags
+   *
+   * @default PropagatedTagSource.NONE
+   */
+  readonly propagateTags?: ecs.PropagatedTagSource;
```

Comment:
The naming on this prop implies that it would be a true/false value in here. I would suggest that be named something like `propagateTagsFrom` or `propagatedTagSource`.